exercises/practice/simple-linked-list/.docs/instructions.append.md

```diff
@@ -1,16 +1,87 @@
 # Instructions append
 
+## How this Exercise is Structured in Python
+
+While `stacks` and `queues` can be implemented using `lists`, `collections.deque`, `queue.LifoQueue`, and `multiprocessing.Queue`, this exercise expects a ["Last in, First Out" (`LIFO`) stack][Baeldung: The Stack Data Structure] (_interactive example [here][LIFO Stack]_) using a _custom-made_ [singly linked list][singly linked list].
+
+<br>
+
+_push and pop happen here_
+
+&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;|
+
+
+(_head_) **`[Node_3]` -> `[Node_2]` -> `[Node_1]`** (_tail_) -> None
```

Comment:
```suggestion
    _push and pop happens here_
                           |
                           |
                           |+----------------------------------\
                                                                |
                                                                v
    +------+    +--------+     +--------+     +--------+    +------+
    | head | ** | Node_3 |---->| Node_2 |---->| Node_1 | ** | tail |
    +------+    +--------+     +--------+     +--------+    +------+

```

Not the best, I am not an artist.  But something like this?

And now I understand what was trying to be communicated, it is the point at which push and pop happen, as opposed to differentiating with push vs shift, etc.

Reply:
I ... don't think that is very readable, and having `head` and `tail` in their own boxes implies that they are nodes, when really `Node_1` is the tail node, and `Node_3` is the head node.